middleware/fibermiddleware_test.go

```diff
@@ -53,6 +55,109 @@ func testMockFiberMiddlewareInvocation(handler fiber.Handler, requestID string,
 
 func TestFiberLogMiddleware(t *testing.T) {
 	hostname := "example.com"
+
+	t.Run("test getHostname with request path without port", func(t *testing.T) {
+		const statusCode = 200
+		const requestID = "my-req-id"
+		const reqPath = "/my-req"
+		var requestPathWithoutPort = fmt.Sprintf("http://%s%s", hostname, reqPath)
+
+		logger, _ := test.NewNullLogger()
+		hook := testMockFiberMiddlewareInvocation(func(c *fiber.Ctx) error {
+			c.Status(statusCode)
+			return nil
+		}, requestID, logger, requestPathWithoutPort)
```

Comment:
@davidebianchi @fredmaggiowski Hi! I can't understand the purpose of this test: it always fail in 404, and I think is because the `requestPathWithoutPort` is not a valid path to be set to the router, and the requests fail.

The bug I'm trying to fix hid this failure, because it caused every request to a non existing route to log a 200 status, so this test didn't fail.

Reply:
I think we can change this test if it does not make sense; I don't recall the specifications for the fiber integration, it was developed by @mattia-fumo maybe he can give us some more insights on this

Reply:
This test just check the hostname parsing and logging behavior in the specific case where the request url has no port specified, why do you think its useless or buggy?

Reply:
I can see it now. I would rather say that we should strip the request path in `testMockFiberMiddlewareInvocation` and register the `app.Get` on it. What do you think?

Reply:
I can see it now. This test needs an hostname in the request, thats why it uses an absolute URL instead of a relative path. Maybe we could always compose a request with an absolute path [here](https://github.com/mia-platform/glogger/pull/37/files#diff-d8990d7a3bab349cf5a6ed0b933a59e463c6632baa708cd7f885c04c9f00dd44R25) to make the call even more realistic.
Like this:
```go
	req := httptest.NewRequest(http.MethodGet, fmt.Sprintf("http://example.com%s", requestPath), nil)
```
And remove [this line](https://github.com/mia-platform/glogger/pull/37/files#diff-d8990d7a3bab349cf5a6ed0b933a59e463c6632baa708cd7f885c04c9f00dd44R63).
Do you agree?

Reply:
Yes I agree, done. I also fixed a bug with the `ip` test in cb80b553031d752aa2405f6988f54a48a7f31e1f, it was declared as empty string, and the test were rewriting on it without using the value after. I just redeclared it as `const` making sure it is not overwritten anywhere.

Reply:
lgtm